### ISSUE TITLE
Reimburse AJ for 2022 Jitsi server

### ISSUE THREAD

**Issue by strugee** [SeaGL/organization#328]
# Request information

Name: AJ Jordan
Date of purchase/expenditure: 11/3/22
Description: DigitalOcean server used to run the 2022 conference's Jitsi server
Reimbursement amount: $72.41
Where the receipt is (e.g. "such-and-such a folder in Google Drive/Nextcloud"; please redact sensitive info): "DigitalOcean Invoice 2022 Nov (1466266-448621017) redacted.pdf" in 2022_SeaGL_Receipts in Google Drive

- [x] I've added a label for my committee in addition to the "Finance" label. If you don't have GitHub permissions to do this, leave this box unchecked.

# Financial committee 

- [ ] Request acknowledged
- [ ] Request reviewed
  - [ ] Approved
  - [ ] Denied
  - [ ] Needs more information
- [ ] Request sent to OSI
- [ ] Reimbursement issued
- [ ] [REQUESTER] Reimbursement received and issue closed

**Comment by NormanGross** [comment]
Hi AJ.  Quick question.   The receipt has a total of $98.38, but you only ask for $72.41.  I wanted to verify that $72.41 is all you are asking for.

**Comment by NormanGross** [comment]
Additionally, please verify amount and update summary tab (Requester Information and Reimbursement Method) of the Google sheet in SeaGL/Finance/2022/Issue 328.....

https://docs.google.com/spreadsheets/d/1uy5is--4b1YR5goncldChGWsajD8n36gRxVOQgePFUs/edit#gid=1740401096

**Comment by strugee** [comment]
> Hi AJ. Quick question. The receipt has a total of $98.38, but you only ask for $72.41. I wanted to verify that $72.41 is all you are asking for.

Yes, sorry - I probably should've been more clear about the receipt. It includes a number of charges unrelated to SeaGL since I use DigitalOcean personally too - I redacted all the personal stuff but I didn't black it out, just deleted it wholesale, so that's why it's kind of hard to read. (So e.g. in the "Droplets" section originally all 5 rows had names in the left-hand column, but I deleted the non-SeaGL names.) $72.41 is the SeaGL-related subset of charges on the receipt.

**Comment by strugee** [comment]
Do you know if any payment methods are easier/harder for OSI?

**Comment by NormanGross** [comment]
> Do you know if any payment methods are easier/harder for OSI?

Nope.   We have had people use both check and PayPal.   no issues.   if you want check, put 'yes' in the check field.  For anything requiring an account, put your account info on the method line.

**Comment by strugee** [comment]
Done, thanks!

**Comment by NormanGross** [comment]
Thank you.  Turned into OSI.
